utility.lua

```diff
@@ -135,23 +145,35 @@ hopper.send_item_to = function(hopper_pos, target_pos, target_node, target_inven
 	local placer = get_placer(hopper_meta:get_string("placer"))
 
 	--target inventory
-	local target_inv = minetest.get_meta(target_pos):get_inventory()
+	local target_inv_name = target_inv_info["inventory_name"]
+	local target_get_inv  = target_inv_info["get_inventory"]
+	local target_inv
+	if target_get_inv then
+		target_inv = target_get_inv(target_pos)
+		if not target_inv then
+			minetest.log("error","No inventory from api get_inventory function: " ..
+				minetest.get_meta(target_pos).name .. " on " .. vector.to_string(target_pos))
```

Comment:
`minetest.get_meta(target_pos).name` will return `nil`. This code does probably not work.
However, `target_node.name` might fit your intention.

Reply:
Thanks!